### PR TITLE
AC-569: Added Toast after User Logout

### PR DESCRIPTION
**Pull request by madhurgupta10**
## Description of what I changed
I added a Toast message to inform user that he/she has successfully logged out.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-569

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by VANKINEENITAWRUN** [comment]
@madhurgupta10 please follow the GuideLines https://wiki.openmrs.org/display/docs/Pull+Request+Tips. The PR should follow the format (see previous merged PRs as an example ) like link to JIRA issue.

**Comment by madhurgupta10** [comment]
@VANKINEENITAWRUN I apologize for not linking the JIRA issue with the PR, I am unable to create a new issue on JIRA that is why I did not ref any issue

**Comment by VANKINEENITAWRUN** [comment]
@madhurgupta10 to be able to create an issue you need to be logged in. Follow this [Link ](https://wiki.openmrs.org/display/docs/Getting+Started+as+a+Developer)before getting Started

**Comment by f4ww4z** [comment]
@madhurgupta10 Please wait until I assess this issue and make it ready for work. See my comment on AC-569. I think that it is unnecessary as the user is taken back to the login page automatically.

File: openmrs-client/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -344,4 +344,5 @@
     <string name="dialog_button_stay">Stay</string>
     <string name="dialog_button_leave">Leave</string>
     <string name="snackbar_nonvisitting_patients">Vitals can only be captured for patients currently in a visit.</string>
+    <string name="logout_success">Logout Success</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="logout_success">Logged out successfully!</string>
```

**Comment by madhurgupta10** [comment]
@f4ww4z Done, thanks for the suggestion :)

**Pull request merged**